core/src/main/res/values/strings.xml

```diff
@@ -331,9 +331,11 @@
 
     <!-- Preferences -->
     <string name="storage_pref">Storage</string>
+    <string name="storage_sum">Managing phone storage, backups, exports...</string>
```

Comment:
Please don't repeat the heading. A "Storage" setting obviously manages storage. This does not provide additional value to the users and clutters the interface.

Reply:
`Episode auto delete, Export, Import`